docker/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM openjdk:11.0.5-jre-slim-buster
+
+COPY orion /opt/orion/
+WORKDIR /opt/orion
+
+# Install libsodium
+RUN set -eux; \
+    apt-get update; \
+    apt-get install -y --no-install-recommends libsodium23=1.0.17-1; \
```

Comment:
what version of libsodium is this installing? We are using libsodium18 in our CI so would be good if this matched.

Reply:
Changed the Docker to use same libsodium18 to be consistent with CI